bonus_guides/G_static_assets.md

```diff
@@ -1 +1,289 @@
+
+#### Brunch Plugin Pipeline
+
+All transformations Brunch performs are actually done by plugins. Brunch automatically uses installed plugins listed in  `package.json`. Here is what the pipeline looks like for a  newly generated Phoenix project:
+
+##### Javascript
+
+- [`babel-brunch`](https://github.com/babel/babel-brunch) transpiles ES6 code to vanilla ES5 Javascript using [Babel](https://github.com/babel/babel)
+- [`javascript-brunch`](https://github.com/brunch/javascript-brunch) is the main Javascript processing plugin. Without it Javascript files will be ignored.
+- [`uglify-js-brunch`](https://github.com/brunch/uglify-js-brunch) minifies Javascript code
+
+##### CSS
+
+- [`clean-css-brunch`](https://github.com/brunch/clean-css-brunch) is a minifier for CSS
+- [`css-brunch`](https://github.com/brunch/css-brunch) is the main CSS processing plugin. Without it  CSS files will be ignored.
+
+
+It is very easy to add more plugins. We can find a plethora of Brunch plugins [here](http://brunch.io/plugins.html) and [here](https://www.npmjs.com/search?q=brunch).
+
+The order in which plugins run is defined by the order in which they appear in `package.json`.
+
+Let's add support for CoffeeScript. Edit `package.json` by adding the following line **before** `javascript-brunch`:
+
+```json
+  "coffee-script-brunch": ">= 1.8",
+```
+
+and run
+
+```
+npm install
+```
+
+Let's rename `greeter.js` into `greeter.coffee` and modify its contents to look like the following:
+
+```coffeescript
+Greet =
+  greet: -> console.log("Hello!")
+
+Bye =
+  greet: -> console.log("Bye!")
+
+module.exports =
+  Greet: Greet
+  Bye: Bye
+```
+
+Once Brunch has built our assets, reload our page, and we should see `Hello!` in the browser Javascript console, just like before.
+
+Adding support for other languages like SASS or React's JSX is as simple as this. Some plugins can be configured in `brunch-config.js`, but they will all work out of the box once installed.
+
+#### Other Things Possible With Brunch
+
+There are many more nice tricks we can do with Brunch which are not covered in this guide. Here are just a few:
+
+- It is possible to have [multiple build targets](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter04-starting-from-scratch.md#split-targets), for example, `app.js` for our code and `vendor.js` for third-party libraries
+- It is possible to control the order of concatenation of files. This might be necessary when working with JS files in `vendor` if they depend on each other.
+- Instead of manually copying third-party libraries into `web/static/vendor` we can [use Bower to download and install them](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter05-using-third-party-registries.md).
+
+
+Should we want one of these, please read  [Brunch documentation](https://github.com/brunch/brunch/tree/master/docs).
```

Comment:
I think we might want a "the" in here.

. . . , please read the [Brunch documentation] . . .